doc/theory/glossary.tex

```diff
@@ -454,7 +457,40 @@
 	\begin{equation*}
 	\langle x, y \rangle = (x_1-y_1)^2 + \cdots + (x_n-y_n)^2.
 	\end{equation*}
-	The associated norm and distance function are referred to as \textit{Euclidean norm} and \textit{Euclidean distance}.  
+	This inner product is referred to as \textit{dot product} and the associated norm and distance function are respectively named \textit{euclidean norm} and \textit{euclidean distance}.	
+	
+	\subsection*{$L^p$-norm:} \label{l^p_norm}	
+	
+	...
```

Comment:
What's the plan for this subsection? To leave it to another PR?

Reply:
It depends on the persistence image entry, TBW concomitantly.

Reply:
I should mention that I included it since it is referenced in persistence landscape.

Reply:
Does the reference still exist?
I am not questioning the need for this section.

Reply:
Yes, the word "p-integrable" should link to L^p